packages/node/test/fixtures/61-edge-function/api/wasm.js

```diff
@@ -0,0 +1,13 @@
+import wasm from '../increment.wasm?module';
```

Comment:
Should we also support `.wat` files? Are those supposed to be executable or just a human-legible representation?

Reply:
These files are not yet supported in Next.js either so we can wait until there’s a requirement for such files I think

Reply:
I wouldn't expect `.wat` to work since no runtime supports them currently that I'm aware of